grafana-plugin/src/components/ScheduleCounter/ScheduleCounter.tsx

```diff
@@ -37,16 +33,16 @@ const ScheduleCounter: FC<ScheduleCounterProps> = (props) => {
       content={
         <div className={cx('tooltip', { [`tooltip__type_${type}`]: true })}>
           <VerticalGroup>
-            <Text type={typeToColor[type]}>{tooltipTitle}</Text>
+            <Text type="secondary">{tooltipTitle}</Text>
             <Text type="secondary">{tooltipContent}</Text>
           </VerticalGroup>
         </div>
       }
     >
-      <div className={cx('root', { [`root__type_${type}`]: true })} onMouseEnter={onHover}>
+      <div className={cx('root', { [`root__type_${type}`]: true }, { padding: addPadding })} onMouseEnter={onHover}>
```

Comment:
_minor_: 'add-padding': addPadding naming looks better because it's clear that it's boolean